inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_sequences_to_sequences_ie.cpp

```diff
@@ -23,14 +23,12 @@ ngraph::pass::ConvertLSTMSequenceMatcher::ConvertLSTMSequenceMatcher() {
             return false;
         }
 
-        const auto& W = std::dynamic_pointer_cast<ngraph::opset4::Constant>(
-                lstm_sequence->input_value(4).get_node_shared_ptr());
+        const auto& W = lstm_sequence->input_value(4).get_node_shared_ptr();
```

Comment:
Why did you remove dynamic pointer cast?

Reply:
in TI to Sequence transformation, we insert an additional Unsqueeze layer, so we have here W->Unsqueeze, not Constant